consensus/types/src/config_and_preset.rs

```diff
@@ -160,7 +160,7 @@ mod test {
             .expect("error opening file");
         let mut mainnet_spec = ChainSpec::mainnet();
         // setting fulu_fork_epoch because we are roundtripping a fulu config
-        mainnet_spec.fulu_fork_epoch = Some(Epoch::new(42));
+        mainnet_spec.gloas_fork_epoch = Some(Epoch::new(42));
```

Comment:
this test was failing because we were setting `mainnet_spec.fulu_fork_epoch` instead of `mainnet_spec.gloas_fork_epoch`, but the check at the end is for `gloas`: `assert_eq!(ConfigAndPreset::Gloas(from), yamlconfig);`